### PR TITLE
Clarify read-only import workflow

### PR DESCRIPTION
**Pull request by turbolay**
#13537

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -172,7 +172,14 @@ Open a text editor and paste the following wallet structure:
 }
 ```
 
-Then paste your _SegWit Extended Account Public Key_ in-between the quotes of the field `"ExtPubKey": "paste segwit xpub here",` and your _Taproot Extended Account Public Key_ in-between the quotes of the field `TaprootExtPubKey": "paste taproot xpub here",`
+Next step depends on your wallet:
+- If it has a _SegWit Extended Account Public Key_, paste it in-between the quotes of the field `"ExtPubKey": "paste segwit xpub here",`
+- If it has a _Taproot Extended Account Public Key_, paste it in-between the quotes of the field `TaprootExtPubKey": "paste taproot xpub here",`
+A wallet must have at least one of those values set to be compatible with Wasabi. It can have both.
```
[the remark this answers is not in the document]
Comment (turbolay) [reply docs/FAQ/FAQ-UseWasabi.md:178]: Please be free to add or change anything you want and merge

**Comment by lontivero** [comment]
The number of stupid things people can do is unlimited and you cannot prevent them from making them.

**Comment by turbolay** [comment]
> The number of stupid things people can do is unlimited and you cannot prevent them from making them.

I can however prevent them from saying I didn't warn

File: docs/FAQ/FAQ-UseWasabi.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md]: 4d7fc6d2470ad32ff8e97d7778179749755b749f

I've made it clear to not use it to send to external wallets.
please check if it's ok for you

**Pull request merged**